main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1997,8 +1997,8 @@ public static void startActivity(final Context context, final String geocode, fi
         context.startActivity(cachesIntent);
     }
 
-    private ActionMode mActionMode = null;
-    private boolean mSelectionModeActive = false;
+    private ActionMode actionMode = null;
+    private boolean selectionModeActive = false;
```

Comment:
If the only changes in a file are tidying ones, and you did not change the code except some names, do not include it in the patch, as it is unrelated to the feature you are trying to introduce.